src/lib/skilling/skills/magic/castables.ts

```diff
@@ -42,5 +62,261 @@ export const Castables: Castable[] = [
 		xp: 55.5,
 		level: 45,
 		ticks: 5
+	},
+	{
+		id: itemID('Bow string'),
+		name: 'Spin Flax',
+		input: new Bank()
+			.add('Astral rune', 1)
+			.add('Nature rune', 2)
+			.add('Air rune', 5)
+			.add('Flax', 5),
+		output: new Bank().add('Bow string', 5),
+		xp: 75,
+		level: 76,
+		ticks: 5,
+		qpRequired: 50
+	},
+	{
+		id: itemID('Green dragon leather'),
+		name: 'Tan Green Dragon Leather',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Nature rune', 1)
+			.add('Fire rune', 5)
+			.add('Green dragonhide', 5),
+		output: new Bank().add('Green dragon leather', 5),
+		xp: 81,
+		level: 78,
+		ticks: 3.5,
+		qpRequired: 50
+	},
+	{
+		id: itemID('Blue dragon leather'),
+		name: 'Tan Blue Dragon Leather',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Nature rune', 1)
+			.add('Fire rune', 5)
+			.add('Blue dragonhide', 5),
+		output: new Bank().add('Blue dragon leather', 5),
+		xp: 81,
+		level: 78,
+		ticks: 3.5,
+		qpRequired: 50
+	},
+	{
+		id: itemID('Red Dragon Leather'),
+		name: 'Tan Red Dragon Leather',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Nature rune', 1)
+			.add('Fire rune', 5)
+			.add('Red dragonhide', 5),
+		output: new Bank().add('Red dragon leather', 5),
+		xp: 81,
+		level: 78,
+		ticks: 3.5,
+		qpRequired: 50
+	},
+	{
+		id: itemID('Black dragon leather'),
+		name: 'Tan Black Dragon Leather',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Nature rune', 1)
+			.add('Fire rune', 5)
+			.add('Black dragonhide', 5),
+		output: new Bank().add('Black Dragon Leather', 5),
+		xp: 81,
+		level: 78,
+		ticks: 3.5,
+		qpRequired: 50
+	},
+	{
+		id: itemID('Gold amulet'),
+		name: 'String Gold',
+		input: new Bank()
+			.add('Astral rune', 2)
```

Comment:
also gives crafting xp in game, same goes for all other String Jewellery below.
which currently is missing.